fabtests/prov/efa/configure.m4

```diff
@@ -0,0 +1,33 @@
+dnl
+dnl SPDX-License-Identifier: BSD-2-Clause OR GPL-2.0-only
+dnl SPDX-FileCopyrightText: Copyright Amazon.com, Inc. or its affiliates. All rights reserved.
+dnl
+dnl Configury specific to the fabtests Amazon EFA provider
+
+
```

Comment:
Suggest save LIBS here and restore to the saved value at the end.  This can limit "-lefa" to only those places needing it.

Reply:
If I keep the LDFLAGS change this shouldn't be needed right? This I won't touch LIBS

Reply:
yes, this shouldn't be needed any more. I was thinking it might be needed by the following AC_CHECK's, but apparently those only check for headers.